assets/css/app/components/_processes_page.scss

```diff
@@ -0,0 +1,88 @@
+.processes-page {
```

Comment:
I am definitely going to reuse this for other tables, so maybe we should give this a generic name to make re-use easier? :)